src/palm-react-native-ui-kit/screens/nft/SendTokenScreen/ConfirmModal.tsx

```diff
@@ -132,84 +156,86 @@ const ConfirmModal = ({
     >
       <View style={styles.body}>
         <View style={{ padding: 20 }}>
-          <View
-            style={{
-              paddingVertical: 12,
-              paddingHorizontal: 16,
-              backgroundColor: '#FF002E0d',
-              borderRadius: 14,
-            }}
-          >
+          <View style={styles.notice}>
             <FormText color={COLOR.error}>
               {t('Nft.SendTokenConfirmModalNotice')}
             </FormText>
           </View>
         </View>
         <View style={styles.itemInfo}>
-          <Row style={{ columnGap: 12, marginBottom: 24 }}>
-            <MoralisErc20Token
-              item={selectedToken}
-              value={UTIL.microfyP(value)}
-            />
-          </Row>
-          <Row
-            style={{
-              position: 'relative',
-              justifyContent: 'center',
-              marginBottom: 12,
-            }}
+          <View
+            style={{ flexDirection: 'column', columnGap: 12, marginBottom: 24 }}
           >
-            <View
-              style={{
-                position: 'absolute',
-                borderWidth: 0.5,
-                borderColor: COLOR.black._200,
-                width: '100%',
-                top: 12,
-              }}
-            />
-            <Row
-              style={{
-                alignItems: 'center',
-                columnGap: 4,
-                marginBottom: 12,
-                backgroundColor: COLOR.black._10,
-                paddingHorizontal: 20,
-              }}
-            >
-              <FormText>{t('Nft.SendTokenConfirmModalOn')}</FormText>
+            <Row style={styles.tokenRow}>
+              <FormText size={24} font="B">
+                {UTIL.addCommasToNumber(value)}
+              </FormText>
+              <View style={styles.token}>
+                <FormImage
+                  source={{ uri: selectedToken.logo ?? images.palm_logo }}
+                  size={20}
+                />
+                <FormText size={20} font="B">
+                  {tokenSymbol}
+                </FormText>
+              </View>
+            </Row>
+            {valueToUsd && (
+              <FormText size={12} color={COLOR.black._400}>
+                {t('Common.UsdPrice', {
+                  price: valueToUsd,
+                })}
+              </FormText>
+            )}
+          </View>
+          <Row style={styles.separateView}>
+            <View style={styles.separateLine} />
+            <Row style={styles.separateRow}>
+              <FormText color={COLOR.black._400}>
+                {t('Nft.SendTokenConfirmModalOn')}
+              </FormText>
               <FormImage source={UTIL.getNetworkLogo(chain)} />
-              <FormText>{_.capitalize(chain)}</FormText>
+              <FormText color={COLOR.black._400}>{`${_.capitalize(chain)} ${
+                isMainnet ? 'MainNet' : 'TestNet'
```

Comment:
I think we can just removed Mainnet vs Testnet indication.

Reply:
removed